### DISCUSSION TITLE
Initial Release

### DISCUSSION THREAD

**Discussion by kuiperzone** [kuiperzone/PupNet-Deploy#4] [Announcements]
INSTALLATION ASSETS
For AppImage - Set execute permission before running. Put file in Path.
For Windows - Launch Command Prompt under Program Menu

+ VERSION 1.0.0; 2023-03-22
- Initial Release
- Tested against: appimagetool: 13 (2020-12-31); rpmbuild: RPM version 4.18.0; dpkg: 1.21.20;flatpak-builder: 1.2.3;InnoSetup: 6.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kuiperzone/PupNet-Deploy/releases/tag/v1.0.0'>Initial Release</a>.</em>

**Comment by mysteryx93** [comment]
Amazing to see how you went beyond the AppImage bash script!

Looking forward to see how it expands in the future for MacOS, Android and iOS support. I'm sure someone will need that.
